index.html

```diff
@@ -4266,9 +4266,10 @@ <h3>Securing Mechanism Specifications</h3>
 [=default graph=].
           </li>
           <li>
-In a [=verifiable presentation=], the property MUST secure the
-[=default graph=] of the [=presentation=] as well as every [=proof graph=]
-related to each [=verifiable credential=] in the [=presentation=].
+In a [=verifiable presentation=], the property MUST secure the 
+[=default graph=] of the [=presentation=], and each 
+[=verifiable credential=], as well as its related [=proof graph=], in the 
+[=presentation=].
```

Comment:
Hmm, something more like this?

```suggestion
In a [=verifiable presentation=], the property MUST secure all
graphs in the [=presentation=], including each 
[=verifiable credential=] and any of its related [=proof graphs=].
```

Reply:
Along the lines of my above, including @dlongley's suggestion --
```suggestion
In a [=verifiable presentation=], the object MUST secure all graphs in
the [=presentation=] not referred to by the object itself, including
each [=verifiable credential=] and any of its related [=proof graphs=].
```

Reply:
@TallTed, I think this should probably say: "the object MUST secure all graphs in
the [=presentation=] not referred to by the property,", right?

Reply:
See my comment above. I'd prefer to close this PR and look at the overall change in a separate PR.